File: overthename/Programmers/level2/숫자의표현.java
class Solution {
	public int solution(int n) {
		int answer = 0;
		int num=0;
		for(int i=1;i<=n;i++){
			int sum=0;
			for(int j=i;i<=n; j++){
				sum+=j;
				if(sum==n){
					num++;
					break;
				}else if(sum>n){
					break;
				}
			}
		}
		return num;
	}
}

File: yunva17/프로그래머스/Lv2/숫자의_표현.js
function solution(n) {
  var answer = 0;

  for (var i = 1; i <= n; i++) {
    var sum = 0;
    for (var j = i; j <= n; j++) {
      sum += j;
      if (sum === n) {
        answer += 1;
        break;
      } else if (sum > n) {
        break;
      }
    }
  }
  return answer;
}
